changelog/1.7.0.md

```diff
@@ -1,21 +1,66 @@
 # Scala.meta 1.7.0
 
+## Cross-compilation against Scala.js
+
+It is our great pleasure to announce that the overwhelming majority of scala.meta modules are now supported by Scala.js!
+In 1.7.0, it is possible to do advanced tasks that include parsing, quasiquoting and prettyprinting in a browser.
+The only unsupported modules are `scalahost` and `testkit`.
+
+## Parser for Scaladoc comments
+
+## Semantic API
+
+- `scalahost` has been split into `scalahost`, a library that provides a scalac-based implementation of `Mirror`,
```

Comment:
I agree, For example. What does `Mirror` provide the user, and how can they use it?
Think of it from the perspective of someone who has never touched meta programming before.